### DISCUSSION TITLE
Paperless-ngx default login

### DISCUSSION THREAD

**Discussion by HMSC2019** [tteck/Proxmox#921] [Help Needed] [answered]
Hi, 
what's the default login for the paperless-ngx docker? Sorry for the question, but I just can't find it

**Comment by tteck** [comment, marked as answer]
It's not Docker.

To Update Paperless-ngx or **Show Paperless-ngx Credentials**, run the following in the LXC Console.
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx.sh)"
```

**Reply by HMSC2019** [reply]
perfect. Thanks

**Reply by itspuia** [reply]
Hi @tteck and thanks for your amazing helper scripts!
Unfortunately running this script in the LXC console doesn't do anything for me. Do I need to look somewhere else for the credentials or am I doing something wrong? 
There is simply no output when I run `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx-v5.sh)"`
Neither an error, nor anything.
Thanks a lot in advance! :)

**Reply by tteck** [reply]
@itspuia that script doesn't exist.

**Reply by itspuia** [reply]
Thanks for the quick reply. But what do you mean? I took it from [your comment](https://github.com/tteck/Proxmox/discussions/921#discussioncomment-4638360). Am I misunderstanding something?

**Reply by tteck** [reply]
The mentioned comment dates back 7 months. It is advisable to always consult the main site for up-to-date links.
https://helper-scripts.com

**Reply by itspuia** [reply]
Thank you very much. I didn't see the "To Update Paperless-ngx..." part on your main page. Sorry for bothering. 
Thanks a lot! :)

**Reply by vednolacni** [reply]
The script does nothing for me. There is simply no output when I run it. Is there any new version?

**Reply by tteck** [reply]
~It seems that you're facing a network problem, probably linked to DNS.~
Always get the newest command from the site.

**Reply by vednolacni** [reply]
I just copied the comand before the instal few minutes ago. Do i need to reinstall it now or just use it again in proxmox shell?

**Reply by kwiat** [reply]
@tteck this comment incl. was helpful for me as well. And thank you for your great collection of scripts. Funny enough, didn't know about this polished website. Through a forum or google and then my browser history always landed on https://tteck.github.io/Proxmox/ (which has the update hint, but I overlooked it as well before) — maybe worth forwarding to the new .com site!

**Comment by tomg1970** [comment]
Hi, what´s the standard login password and username?
admin:admin i have tested

**Reply by tteck** [reply]
Kidding right?

**Reply by tomg1970** [reply]
Don't laugh. I am serious. I have installed the installation script from
[](https://tteck.github.io/Proxmox/) installed. And if I get myself to the WebUI with
the IP address to the WebUI, it should be according to the script user: admin 
and password: admin?
Or do I see something completely wrong

**Reply by tteck** [reply]
Here's your answer to **Show Paperless-ngx Credentials**, https://github.com/tteck/Proxmox/discussions/921#discussioncomment-4638360

**Reply by tomg1970** [reply]
Yes, I have done that:
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx-v5.sh)"`

Then comes this error:
`cat: paperless.creds: No such file or directory
[ERROR] in line 393: exit code 1: while executing command cat paperless.creds`

**Reply by tteck** [reply]
> according to the script user: admin and password: admin

Where do you see this?

**Reply by tomg1970** [reply]
Now it worked with the scipt. That only works after a reboot.
Then I also found the credentials via the credentials.
Thanks for your super fast feedback

**Comment by maggos1211** [comment]
Hi,

after i update with bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx-v5.sh)" in the paperless console, i cant log in. I see my credentials, but this dosn´t work anymore. Standard password is also not working. I restored the Backup many times, and test it with a new user, this password is also false.

**Reply by maggos1211** [reply]
PAPERLESS_AUTO_LOGIN_USERNAME=<username> doesn´t work
PAPERLESS_ADMIN_USER=<username> doesn´t work
PAPERLESS_ADMIN_PASSWORD=<password>

I´m the only one with this problem?

Thank you

**Reply by vino-tinto** [reply]
Hi maggos1211,
I found the solution.
Look in  /root/paperless.creds .
Here you will find all User and Passwords.

Good luck!

**Reply by maggos1211** [reply]
Its the same as in the credentials when you go over the bash -c ....

But i can see, after the update, the paperless-ngx folder in root is **gone**.

**Reply by tteck** [reply]
Did you install Paperless-ngx using the script in this repository? I ask because the directory path has always been `/opt/paperless`.

**Reply by maggos1211** [reply]
`root@paperless-ngx:~# cd /opt/paperless/`
`root@paperless-ngx:/opt/paperless# ls`
`Dockerfile  Pipfile       README.md    consume  docker  gunicorn.conf.py  paperless.conf    scripts  static`
`LICENSE     Pipfile.lock  __pycache__  data     docs    media             requirements.txt  src`
`root@paperless-ngx:/opt/paperless# cd /root/paperless-ngx/`
`root@paperless-ngx:~/paperless-ngx# ls`
`Dockerfile  Pipfile       README.md  data    docs              media           requirements.txt  src`
`LICENSE     Pipfile.lock  consume    docker  gunicorn.conf.py  paperless.conf  scripts           static`

I will test it in a new installation on sunday. i would say yes to your question, but i test many of your scripts in the last days. I can´t really answer :)

**Reply by tteck** [reply]
`/root/paperless-ngx/` should not exist. My guess is that there has been some experimenting and things got messed up.

**Comment by vino-tinto** [comment]
Hi,
in order to get the credentials I run  bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx-v5.sh)" .
The results are:
 ✓ Stopped Paperless-ngx
 ✓ paperless-task-queue.service Exists.
 ✓ Updated to v1.13.0
 ✓ Cleaned
 ✓ Started Paperless-ngx
 ✓ Updated Successfully!
When I open the WebUI the fields for username and password are empty.
What do I do wrong? How do I get the credentials?

**Reply by letsruletheworld** [reply]
this only works if the lxc is installed with default values.  i changed the installation options with the installer - and afterwards I could not get into the shell, it was asking for loging credentials at shell level.

**Reply by tteck** [reply]
Use the password that you supplied in Advanced Settings

**Reply by letsruletheworld** [reply]
ok - i might have misssed that.

why not printing out the credentials directly after installation?  in log it is still mentioning that it generates the login values.

**Comment by HolgerM1** [comment]
Hi,
ich hänge mich hier mal mit meinem Problem dran. Ich habe paperless mit Deinem script installiert. Leider kann ich das gui nicht erreichen. Ich bekomme bei der Eingabe von IP:8000 immer die Fehlermeldung "Verbindung fehlgesschlagen" (in Firefox) respektive "connection refused" in Chrome?!

**Reply by tteck** [reply]
I'm sorry to hear that you're experiencing difficulties. To better assist you, can you provide more information, such as whether the script completed successfully or not, and if you have attempted to reinstall?

**Reply by HolgerM1** [reply]
Hi, sorry for witing in german. I guess I was to impatient. I didn't expect the installation taking that much time. It's working now, thanks for your fast response!

**Reply by HolgerM1** [reply]
I have to come back on this again. Neither admin:admin nor root:PW(set during installation for ssh) are working for login via gui. I restarted the server without success?!

**Reply by tteck** [reply]
You can find the instructions to **show Paperless-ngx login credentials** in the same place where you found the installation command.

**Reply by tteck** [reply]
Or, read this entire thread.

**Reply by HolgerM1** [reply]
Oh, sorry. Sometimes it's better to read, than to fly over the instructions :-)

To my appologize, never seen a software that makes initial setup so complicated....

**Reply by tteck** [reply]
By following the provided instructions carefully, the process should not be too complicated.

**Reply by HolgerM1** [reply]
of corse, Thank you very much for all the scripts! Working amazing!

**Reply by HolgerM1** [reply]
If I may ask another question. i'm comming from another DMS and would like to continue using an Input folder I can share in the network. I found [this ](https://www.schreiners-it.de/proxmox/paperless-ngx-im-lxc-container-installieren/)article, but trying to mount the folder from my Windows maschine does not work. I tried "IP/paperless/" and the gui login. Does not work.

**Comment by jase31** [comment]
Im getting a similar problem. Ive installed paperless using the helper. Console now wants me to enter username/password which I dont yet have. I cant runt the command to get the credentials (below) because the console screen is waiting for user/name and password. 

How can I enter the command into console? when it is waiting for a password?

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx.sh)"

**Reply by tteck** [reply]
❗ It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.
